feedback/static/html/feedback_instructor.html

```diff
@@ -26,7 +26,7 @@
             <td colspan="6" role="cell" class="pgn__data-table-cell-wrap">
               <a class="admin-panel" href="{{ block.url }}"><button class="btn btn-primary" ="button">{% trans "See Unit" %}: {{ block.parent }}</button></a>
             </td>
-            <td colspan="3" role="cell" class="pgn__data-table-cell-wrap">
+            <td colspan="6" role="cell" class="pgn__data-table-cell-wrap">
               <ul>
                 {% for vote_aggregate in block.vote_aggregate %}
                 <li>{{ vote_aggregate.scale_text }}: {{ vote_aggregate.count }}</li>
```

Comment:
Could you please set style attribute of this `<li>` to `white-space: nowrap` to avoid wrapping of text?

Reply:
Yep, I can add it but the problem with that is if the `vote_aggregate.scale_text` is too long won't appear full text
<img width="663" alt="Screen Shot 2024-03-20 at 8 47 19 AM" src="https://github.com/openedx/FeedbackXBlock/assets/134975835/18653209-5fb5-42fe-94b3-bef311fee8fe">